src/main/java/frc/robot/subsystems/intake/commands/Commandy.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.intake.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Intake;
+
+/**
+ * Add Commandy function;
+ */
+public class Commandy extends CommandBase {
+    private final Intake intake;
+    private final double power;
+
+    /**
+     * Add Requirements
+     * @param intake name to Intake.
+     */
+    public Commandy(Intake intake, double power) {
+        this.intake = intake;
+        this.power = power;
+        addRequirements(intake);
+
+    }
+
+
+    @Override
+    public void initialize() {
+        super.initialize();
```

Comment:
No need for this call to `super.` and also the ones in the methods following. As you can see in WPILib's source code, these methods are empty in the parent class, so these calls do nothing (source code taken from [here](https://github.com/wpilibsuite/allwpilib/blob/d8e0b6c9775fcb1b9224df840c6e824a673e32d4/wpilibNewCommands/src/main/java/edu/wpi/first/wpilibj2/command/Command.java#L18-L44)):
```java
public interface Command {
  /** The initial subroutine of a command. Called once when the command is initially scheduled. */
  default void initialize() {}


  /** The main body of a command. Called repeatedly while the command is scheduled. */
  default void execute() {}


  /**
   * The action to take when the command ends. Called when either the command finishes normally, or
   * when it interrupted/canceled.
   *
   * <p>Do not schedule commands here that share requirements with this command. Use {@link
   * #andThen(Command...)} instead.
   *
   * @param interrupted whether the command was interrupted/canceled
   */
  default void end(boolean interrupted) {}


  /**
   * Whether the command has finished. Once a command finishes, the scheduler will call its end()
   * method and un-schedule it.
   *
   * @return whether the command has finished.
   */
  default boolean isFinished() {
    return false;
  }
```